javascript/ecmascript-2015/number-extensions/number-extensions-part-1.md

```diff
@@ -48,15 +46,15 @@ Number.isInteger(0.1); //false
 
 ### Number.isNaN
 
-Before ES6 it was difficult to test if a value was equal to *NaN* (Not a number). This is because NaN == NaN evaluates to false.
+Before ES6 it was difficult to test if a value was equal to `NaN` (Not a number). This is because `NaN == NaN` evaluates to false.
 
-Whilst a global *isNaN* function has existed in previous versions it has the issue that it converts values which makes it hard to test if something is really *NaN*:
+Whilst a global `isNaN` function has existed in previous versions, it converts the given value to a number before checking if the given number is equal to `NaN`, making it hard to test if something is really `NaN`:
```

Comment:
@kapnobatai137 Worth adding that `!Number.isNan(<value>)`, Number.parseFloat(<value>) is not a reliable way of checking if a value is number or not.

Because `!Number.isNaN("")` is `true`. And while `Number.parseFloat("")` does give `NaN` which can be checked via `Number.isNaN`, it fails on input like `"1.1.1"`

One reliable way to check if a given string is a valid number is `const isNum = num => num !== "" && Number(num) === Number(num)`

Reply:
I think the 

" One reliable way to check if a given string is a vali .." -> can go in insight text

rest in the footnotes.

thoughts?